bucket/armcord.json

```diff
@@ -0,0 +1,26 @@
+{
+  "homepage": "https://github.com/ArmCord/ArmCord",
+  "description": "ArmCord is a custom client designed to enhance your Discord experience while keeping everything lightweight.",
+  "version": "3.0.5",
+  "license": "OSL-3.0",
+  "architecture": {
+    "64bit": {
+      "url": "https://github.com/ArmCord/ArmCord/releases/download/v3.0.5/ArmCord-Setup-3.0.5.exe#/armcord-3.0.5.7z",
+       "hash": "cc8391f98139195ea2ab79aa9ad3e439b9bc1eceb9a32cfe67788c38e4b0638c",
+        "pre_install": [
+                "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\"",
+                "Remove-Item \"$dir\\`$*\", \"$dir\\Uninst*\" -Recurse"
+            ]
+    }
+  },
+  "bin": "ArmCord.exe",
+  "checkver": "github",
+  "autoupdate": {
+    "architecture": {
+      "64bit": {
+        "url": "https://github.com/ArmCord/ArmCord/releases/download/v$version/ArmCord-Setup-$version.exe#/armcord-$version.7z",
+         "hash": "cc8391f98139195ea2ab79aa9ad3e439b9bc1eceb9a32cfe67788c38e4b0638c"
```

Comment:
`autoupdate` -> `hash` is for "checksum" files (files which contains the hash value of the latest version), or if the developer writes hash on their page.